source/index.rst

```diff
@@ -2,242 +2,53 @@ Accessibility
-
-The SPEC process is designed to identify areas of shared concern between
-projects in the scientific Python ecosystem and to produce
-collaboratively written, community adopted guidelines for addressing
-these areas. Such guidelines are known as ‘SPEC documents’ or a ‘SPEC’.
-
-Projects in the ecosystem have an existing, diverse set of proposal
-processes and development constraints. SPECs complement these: they are
-a mechanism to encourage shared practices and improve uniformity of
-experience across projects. This SPEC captures established practices so
-that new projects can learn from them. As SPECs are living documents,
-authors may propose a new practice that they believe will benefit the
-ecosystem as a whole.
-
-Projects decide for themselves whether to adopt any given SPEC—often,
-this would be through team consensus. A SPEC may not be a good fit for
-every single project, and thus there is no expectation that all SPECs
-must be adopted by all projects. That said, SPECs serve their purpose
-through being adopted by several projects—and their authority stems from
-the extent to which they are. [citation, SPEC Processes page]
-
-The purpose of this SPEC document is to: - Make the ecosystem more
-accessible - Clarify the broad term accessibility and its various
-possible implementations - Provide specific assessment tools and steps -
-Direct to existing accessibility standards and modifications for open
-source specifically - Provide case studies of how different core
-projects became more accessible - Start discussion and develop shared
-solutions
+   image-descriptions
+
+
+Introduction
+------------
+
+`Accessibility <https://en.wikipedia.org/wiki/Accessibility>`_ is building 
+things that are fully usable for people with disabilities. `There are 
+many reasons to prioritize accessibility <https://www.w3.org/WAI/fundamentals/accessibility-intro/#important>`_ , 
+including ethical, user experience, and legal motivations. Accessibility 
+is also a long-recommended web standard (via `WCAG <https://www.w3.org/TR/WCAG/>`_).
+
+Refer to the `draft SPEC associated with this effort <https://scientific-python.org/specs/spec-0003/>`_
+for more information.
+
+Global accessibility guidelines
+-------------------------------
+
+There are accessibility guidelines that extend beyond the Scientific Python 
+ecosystem; we respect those first and foremost. This includes
+
+* `Web Content Accessibility Guidelines (W3C) <https://www.w3.org/TR/WCAG/>`_
+
+Please note that in many cases guidelines serve as a foundation for 
+accessibility efforts. They are by no means the limit of what is needed to 
+create enjoyably accessible work.
+
+Scientific Python recommendations
+---------------------------------
+
+For cases where ecosystem-specific recommendations are needed or examples of 
+accessibility within the ecosystem are helpful, we also provide resources 
+designed to support the needs of our community.
+
```

Comment:
```suggestion
If you would like to add, edit or suggest new recommendations, join the community at [accessibility.scientific-python.org](accessibility.scientific-python.org/community.html)
```

Reply:
Added to match the [suggestion in the other PR for the main SPEC website](https://github.com/isabela-pf/specs/pull/2/files#r865339886).